controllers/openstack_ansibleee_controller.go

```diff
@@ -137,6 +160,19 @@ func (r *OpenStackAnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.R
 		instance.Status.NetworkAttachments = map[string][]string{}
 	}
 
+	if instance.Status.Conditions.IsFalse(redhatcomv1alpha1.AnsibleExecutionJobReadyCondition) {
+		if foundJob.Status.Succeeded > 0 {
+			r.Log.Info(fmt.Sprintf("%s ready", redhatcomv1alpha1.AnsibleExecutionJobReadyCondition))
+			instance.Status.Conditions.Set(condition.TrueCondition(
+				redhatcomv1alpha1.AnsibleExecutionJobReadyCondition,
+				redhatcomv1alpha1.AnsibleExecutionJobReadyMessage))
+		} else {
+			r.Log.Info(fmt.Sprintf("%s not yet ready, requeueing", redhatcomv1alpha1.AnsibleExecutionJobReadyCondition))
+			return ctrl.Result{RequeueAfter: time.Second * 2}, nil
```

Comment:
I don't think the requeue is actually required here b/c it's setup to watch the owned jobs in SetupWithManager already.